### PR TITLE
Fix uplink detection (different interfaces for Kathleen and Hedy)

### PR DESCRIPTION
**Pull request by akira25**
This PR solves issue #2.

### CODE REVIEW

File: owm2ffmap/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
-tornado
```
Comment (SvenRoederer) [inline owm2ffmap/requirements.txt]: Can you put this in a separate commit, as it don't seem to relate to the "uplink-issue" fixed

**Comment by SvenRoederer** [comment]
Thanks for adding the python-doc, but can you put this into a separate commit to not mix two very different things in one commit.

File: owm2ffmap/owm2ffmap.py [hunk 2]
```diff
@@ -170,6 +177,7 @@ def process_node_json(comment, body, hostid=None, firstseen=None, lastseen=None)
         if not check_location(float(longitude), float(latitude)):
             print("...out of geographic bounds, skipping")
             return
+        # check if the router itself has an uplink. returns True or False
         isuplink = len([a for a in owmnode.get("interfaces", []) if a.get("ifname", "none") == "ffvpn"]) > 0
```
Comment (SvenRoederer) [inline owm2ffmap/owm2ffmap.py]: Can you move your Hedy-uplink-test close to this line? Keeps relating code close to each other.

File: owm2ffmap/owm2ffmap.py [hunk 3]
```diff
@@ -32,9 +32,12 @@
 firmware_kathleen1505 = re.compile("^Freifunk Berlin kathleen 15.05(\.1){0,1}$")
 firmware_openwrt = re.compile("^OpenWrt .*")
 
+hedy_match = re.compile("v1.[0-9].[0-9]")
```
Comment (SvenRoederer) [inline owm2ffmap/owm2ffmap.py]: can you reuse the "firmware_hedy" regex we already have defined?

**Comment by akira25** [comment]
Thank you for the advice. :-) I think I met them all.

**Comment by akira25** [comment]
I think #7 needs to be merged first. But im not quite sure about that...

File: owm2ffmap/owm2ffmap.py (hunk 2 above)
Comment (SvenRoederer) [reply owm2ffmap/owm2ffmap.py]: thanks, looks much more nice now.

**Comment by SvenRoederer** [comment]
Can you rebase on master and drop the first 2 commits? Then it will apply perfectly.

**Comment by akira25** [comment]
@SvenRoederer I did rebase the commit as asked for. It's mergeable now.

**Comment by SvenRoederer** [comment]
I would like to give the final decision to @sarumpaet , as he knows the server-setup and running-code the best.
He has also to trigger the the git pull on the machine.
